index.html

```diff
@@ -12,6 +12,23 @@
 </head>
 
 <body>
+    <nav class="navRow">
+        <ul>
+            <li id="logo"><img src="img/lambda-black.png">  </li>
+            
```

Comment:
Don't forget to make these anchor elements so the user can go to the about page and back to the home page.